server/channels/app/platform/service.go

```diff
@@ -353,9 +353,7 @@ func (ps *PlatformService) Start() error {
 		message := model.NewWebSocketEvent(model.WebsocketEventConfigChanged, "", "", "", nil, "")
 
 		message.Add("config", ps.ClientConfigWithComputed())
-		ps.Go(func() {
-			ps.Publish(message)
-		})
+		ps.Publish(message)
```

Comment:
I thought the goroutine here was purely a defense against potential deadlocks in case the listener gets somehow called synchronously from within the hub handler. But at first glance I couldn't find an execution path that would lead there, plus we use a buffered channel so it would be very edge case to begin with.